packages/hooks/tests/useNFTViewMethod.test.ts

```diff
@@ -0,0 +1,27 @@
+import { renderHook, cleanup } from '@testing-library/react-hooks'
```

Comment:
Test here seems to fail:

```
 FAIL  tests/useNFTViewMethod.test.ts
  ● useNFT › should get data from blockchain without any error
    expect(received).toBeUndefined()
    Received: [Error: [-32000] Server error: account 0.share-nft.testnet does not exist while viewing]
  ```

Reply:
This is because you switched to using `mainnet` network and this test is used to test a call from `testnet`. 

This should be solved with this issue #95